langchain/src/document_loaders/web/notionapi.ts

```diff
@@ -11,44 +15,118 @@ import type {
   MdBlock,
 } from "notion-to-md/build/types";
 
-import { BaseDocumentLoader } from "../base.js";
+import Bottleneck from "bottleneck";
 import { Document } from "../../document.js";
+import { BaseDocumentLoader } from "../base.js";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type GuardType<T> = T extends (x: any, ...rest: any) => x is infer U
   ? U
   : never;
 
+type GetBlockResponse = Parameters<typeof isFullBlock>[0];
+type GetPageResponse = Parameters<typeof isFullPage>[0];
+type GetDatabaseResponse = Parameters<typeof isFullDatabase>[0];
+
 type BlockObjectResponse = GuardType<typeof isFullBlock>;
 type PageObjectResponse = GuardType<typeof isFullPage>;
+type DatabaseObjectResponse = GuardType<typeof isFullDatabase>;
+
+type GetResponse =
+  | GetBlockResponse
+  | GetPageResponse
+  | GetDatabaseResponse
+  | APIResponseError;
+
+// const isBlockResponse = (res: GetResponse): res is GetBlockResponse =>
```

Comment:
Remove commented out methods

Reply:
Sure, I left it in as it's a useful reference for anyone wanting to use the notion SDK in the future, but this isn't the place for it. I'll submit a PR to notion-sdk to deal with types more succinctly.

Reply:
I have removed the unused methods.